### PR TITLE
Add examine for caustic and cold damage

### LINKED ISSUE #29991: Caustic damage is not examinable
## Description
Despite visibly scorching damaged humanoids like heat damage, game won't tell you cause of damage is caustic.

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

title
resolves #29991 
## Why / Balance

qol

## Media

Screenshot_20240713_105500

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added health examine for caustic and cold damage.

### CODE REVIEW

**Comment by Plykiya** [comment]
the correct grammar would be "frostbite", not "frostbites"
good PR

File: Resources/Locale/en-US/health-examinable/health-examinable-carbon.ftl [hunk 1]
```diff
@@ -15,4 +15,8 @@ health-examinable-carbon-Heat-25 = [color=orange]{ CAPITALIZE(SUBJECT($target))
 health-examinable-carbon-Heat-50 = [color=orange]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } major burns across { POSS-ADJ($target) } body.[/color]
 health-examinable-carbon-Heat-75 = [color=orange]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } severe third-degree burns across { POSS-ADJ($target) } body![/color]
 
+health-examinable-carbon-Cold-25 = [color=lightblue]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } minor frostbite across { POSS-ADJ($target) } body.[/color]
+health-examinable-carbon-Cold-50 = [color=lightblue]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } major frostbite across { POSS-ADJ($target) } body.[/color]
+health-examinable-carbon-Cold-75 = [color=lightblue]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } severe third-degree frostbite across { POSS-ADJ($target) } body![/color]
```
Comment (deltanedas) [inline Resources/Locale/en-US/health-examinable/health-examinable-carbon.ftl:20]: frostbite is cell rupturing from ice crystals or something i dont think third degree applies

**Comment by thebadman4662** [comment]
Ah yes, I believe I never made issue post for caustic damage so if its not too much, https://github.com/space-wizards/space-station-14/issues/29991 could also get included?

**Comment by lzk228** [comment]
updated

File: Resources/Locale/en-US/health-examinable/health-examinable-carbon.ftl (hunk 1 above)
Comment (Plykiya) [reply Resources/Locale/en-US/health-examinable/health-examinable-carbon.ftl:22]: "Severity may be divided into superficial (1st and 2nd degree) or deep (3rd and 4th degree)"
https://en.m.wikipedia.org/wiki/Frostbite

**Review by Tayrtahn — APPROVED** [review]

**Pull request merged**